ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/get_anta_results.py

```diff
@@ -159,7 +159,7 @@ def load_custom_catalogs(catalog_files: list[Path]) -> dict:
             with file.open("r", encoding="UTF-8") as fd:
                 catalog = load(fd, Loader=CSafeLoader)
                 catalog_list.append(catalog)
-        except (OSError, YAMLError) as error:
+        except (OSError, YAMLError) as error:  # noqa: PERF203
```

Comment:
probably worth adding what we are noqa-ing with a comment here as this can be none obvious when reviewing. But also looking at the code, we will raise on the first failure anyways so we can put the `try` around the `for` loop

Reply:
I tried moving it out, but it did now work since the vars are not available.